src/generic/ConnectionErrorAlert.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { injectIntl, intlShape, FormattedMessage } from '@edx/frontend-platform/i18n';
+import { Alert } from '@edx/paragon';
+import { getConfig } from '@edx/frontend-platform';
+
+import messages from '../messages';
+
+function ConnectionErrorAlert({ intl }) {
+  return (
+    <Alert variant="danger" data-testid="connectionErrorAlert">
+      <FormattedMessage
+        id="authoring.alert.error.connection"
```

Comment:
@davidjoy Not sure if I have asked this before, but can you please explain when to use `FormattedMessage` component for intel. and when to use messages in `messages.js` with the syntax `{intl.formatMessage(messages.supportText)}`?

Reply:
There are few differences.  One of them, though, is that the `FormattedMessage` component can accept JSX as a variable substitution.  In this case, we're using that to add an `<Alert.Link>` component around the link to the support page.  If we tried this with `formatMessage`, it displays `[Object object]` instead of the desired link.

You'll see folks using `FormattedMessage` even if they don't need the above - for some reason I prefer the messages file because it leaves a smaller footprint in the component code, but we don't have any guidance one way or the other.  FormattedMessage is nice because it shows you the English text right there in your component, which can aid comprehension.